src/Routing.tsx

```diff
@@ -18,14 +19,22 @@ const routes = [
     path: 'oops',
     element: OopsPage,
   },
+  {
+    path: 'upcoming',
```

Comment:
My guess is we'd want the path to be "roadmap" - same with the names of other components that are related.

Reply:
You mean renaming all the stuff which contains Upcoming to Roadmap (components, folders, everything)? I would rather to this in different PR (I can prepare) - in this I just changed the path to be `/roadmap`. I think it will be a lot of stuff

Reply:
That's fine with me!